### DISCUSSION TITLE
Form doesn't appear

### DISCUSSION THREAD

**Discussion by groupe8chiffres** [rustdesk/rustdesk#9009] [Q&A]
![RustDesk A](https://github.com/user-attachments/assets/d4bd4eef-a1db-415f-b8f9-b3b0b6edb98a)

I connect to distant computer, every thing is correct it is like if I was there but the image of RustDesk on the right corner appear white with nothing on it. It is the same if I run a program where there is form. White page only with title. It seam like, if there is a window open over the main screen, RustDesk doesn't see the writing. If I run Excel every thing is correct but not database. 

![RustDesk A](https://github.com/user-attachments/assets/13e8bb76-2545-4f88-944e-48ef1566021f)

**Comment by rustdesk** [comment]
a screenshot video?

**Reply by groupe8chiffres** [reply]
I insert a picture now
